src/Camera.cc

```diff
@@ -105,22 +66,23 @@ void Camera::setProjection(ProjectionType type)
 
 void Camera::resetView()
 {
-	type = CameraType::GIMBAL;
-	object_rot << 35, 0, -25;
-	object_trans << 0, 0, 0;
-	viewer_distance = 140;
+	setVpr(55, 0, 25);  // set in user space units
+	setVpt(0, 0, 0);
+	setVpd(140);
 }
 
-double Camera::zoomValue()
+double Camera::zoomValue() const
 {
 	return viewer_distance;
 }
 
-std::string Camera::statusText()
+std::string Camera::statusText() const
 {
+	auto vpt = getVpt();
+	auto vpr = getVpr();
```

Comment:
You could 'const' both these for a bit extra readability and compile-time checks.
You could even make those methods return a `const &`, but that's just optimization which probably doesn't give us much here.

Reply:
OK